### ISSUE TITLE
Cannot connect to the pool

### ISSUE THREAD

**Issue by LycheMiner** [UnamSanctam/SilentCryptoMiner#1850]
Why do some machines write that it is impossible to connect to the pool? pool![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/147152975/84c63033-d080-4683-8f76-c5fcc0511152)supportxmr.com:9000

**Comment by UnamSanctam** [comment]
Do you have SSL/TLS enabled? Since that's the SSL port. And SupportXMR is also well known for blocking a lot of IP addresses and also for being blocked in some countries and ISPs.

**Comment by LycheMiner** [comment]
Yes. of course included, on most machines all good and sometimes it happens that everything works works and suddenly writes a connection error, or immediately with an error go in

**Comment by LycheMiner** [comment]
What pool should I use to avoid IP blocking?

**Comment by UnamSanctam** [comment]
2miners or nanopool usually.
